lib/wallet/test/unit/SpecHook.hs

```diff
@@ -0,0 +1,6 @@
+module SpecHook where
```

Comment:
I've noticed there is already `lib/test-utils/test/Test/Hspec/ExtraSpec.hs` for a similar purposes, do you think it makes sense to move this utility there?

Reply:
I suspect that module is to help with integration tests. So perhaps we could leave it untouched for the moment.